test/metric_learn_test.py

```diff
@@ -166,8 +166,7 @@ def test_deprecation_num_labeled(self):
       # test that a deprecation message is thrown if num_labeled is set at
       # initialization
       # TODO: remove in v.0.6
-      X = np.array([[0, 0], [0, 1], [2, 0], [2, 1]])
-      y = np.array([1, 0, 1, 0])
+      X, y = make_classification()
```

Comment:
The test was failing before so I changed it, I think this is reasonable since what was before was not realistic